parts/kubernetesmastercustomscript.sh

```diff
@@ -200,8 +200,30 @@ function configNetworkPolicy() {
     fi
 }
 
+function systemctlEnableAndCheck() {
+    systemctl enable $1
+    systemctl is-enabled $1
```

Comment:
That's only because the install failed and we weren't failing this script.

I don't really think this is necessary... Instead we should be setting `set -e` in this script so that any uncaught/unhandled errors just fail the entire script.